<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportElementDialog</class>
 <widget class="QDialog" name="ImportElementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intégration d'un élément</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>L'élément a déjà été intégré dans le projet. Toutefois, la version que vous tentez de poser semble différente. Que souhaitez-vous faire ?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="m_use_actual_rd">
     <property name="text">
      <string>Utiliser l'élément déjà integré</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="m_use_drop_rb">
     <property name="text">
      <string>Intégrer l'élément déposé</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QRadioButton" name="m_erase_actual_rb">
       <property name="text">
        <string>Écraser l'élément déjà intégé</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="m_use_both_rb">
       <property name="text">
        <string>Faire cohabiter les deux éléments</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportElementDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportElementDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_use_drop_rb</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_erase_actual_rb</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_use_drop_rb</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_use_both_rb</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
